neqo-transport/src/recovery/mod.rs

```diff
@@ -321,13 +321,15 @@ impl LossRecoverySpace {
         );
         self.first_ooo_time = None;
 
-        let largest_acked = self.largest_acked;
+        let Some(largest_acked) = self.largest_acked else {
```

Comment:
It'd be good to add some comments here to explain this.

Reply:
This got me thinking about whether the logic here got broken, causing the performance issue. I have a possible fix in 76c32f3101e61b6c8864a44a70413321a15d1475; let's say what CI says.

Reply:
The `transfer.rs` benchmarks are back to normal.

> transfer/pacing-false/varying-seeds: No change in performance detected.
> transfer/pacing-true/varying-seeds: No change in performance detected.

But the client/server transfer benchmarks are still elevated.

```
neqo 	neqo 	reno 	on 	456.7 ± 51.6 	375.9 	567.0 	1.00
neqo 	neqo 	reno 		262.9 ± 126.9 	178.7 	620.9 	1.00
neqo 	neqo 	cubic 	on 	450.3 ± 58.1 	370.8 	580.7 	1.00
neqo 	neqo 	cubic 		214.4 ± 28.5 	182.3 	280.3 	1.00
```